source/guides/faq.md

```diff
@@ -49,6 +49,55 @@ See <https://github.com/nix-community/home-manager>
 
 ## NixOS
 
+### How to run non-nix executables?
+
+NixOS cannot run dynamically linked executables intended for generic linux environments out of the box.
+This is because, by design, it does not have a global library path, nor does it follow the traditional linux standard for where program files are stored on the filesystem.
+
+:::{note}
+The NixOS community uses the term "FHS" for traditional linux environments, which is short for "[Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html)".
+:::
```

Comment:
```suggestion
This is because, by design, it does not have a global library path, nor does it follow the [Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html) (FHS).
```

There is nothing special about the Nix community using that abbreviation, and mentioning it should be enough.

Reply:
We're the only ones who commonly use the term at all, though, abbreviated or not. So it almost certainly needs to be introduced as new jargon to those reading this. I'm not really comfortable using the word here without a prominent introduction to what it means.

I'm a bit on the fence about whether it's prominent enough without the note. It's certainly shorter and more straightforward, which is nice, but not everyone reads linearly, and I want the introduction of the term to catch the eye.

Reply:
Yeah, alright, I'm good with it. The different coloring of the link is enough, I think.